.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -0,0 +1,40 @@
+name: 📚Documentation
```

```suggestion
name: 📚 Documentation
```

there is a space between emoji and 'Documentation'

---

@chaso1495 This one should be updated :) Check this comment, please 😸

---

@yeonns I checked and fixed it :)